cscore/src/main/java/edu/wpi/cscore/UsbCamera.java

```diff
@@ -43,4 +43,13 @@ public UsbCamera(String name, String path) {
   public String getPath() {
     return CameraServerJNI.getUsbCameraPath(m_handle);
   }
+
+  /**
+   * Set how verbose the camera connection messages are.
```

Comment:
Doxygen typically separates the description and the @ tags with an empty line.

Reply:
We have this in other comments though, so I'd prefer to postpone doing that to a comment cleanup PR.